### PR TITLE
Added tel codes seeding

### PR DESCRIPTION
**Pull request by d-mv**
Added telephone codes seed to `seeds.rb`

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,15 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# url to codes json
+url = 'https://gist.githubusercontent.com/Goles/3196253/raw/9ca4e7e62ea5ad935bb3580dc0a07d9df033b451/CountryCodes.json'
```
Comment (knudsem) [inline db/seeds.rb:10]: Nice Dimi !

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,15 @@
...
+# get the file and parse it
+codes = JSON.parse(open(url).read)
```
Comment (knudsem) [inline db/seeds.rb]: are these country codes? could we change to country_codes.

**Review by knudsem — COMMENTED** [review]
Happy, lets just change that variable :)

File: db/seeds.rb (hunk 1 above)
Comment (d-mv) [reply db/seeds.rb:10]: Thanks

File: db/seeds.rb [hunk 3]
```diff
@@ -9,9 +9,9 @@
 # url to codes json
 url = 'https://gist.githubusercontent.com/Goles/3196253/raw/9ca4e7e62ea5ad935bb3580dc0a07d9df033b451/CountryCodes.json'
 # get the file and parse it
-codes = JSON.parse(open(url).read)
+country_codes = JSON.parse(open(url).read)
 # go through the elements
-codes.each do |code|
+country_codes.each do |code|
```
Comment (knudsem) [inline db/seeds.rb:14]: Looking good mamen!

**Review by knudsem — APPROVED** [review]
I can read it better

**Pull request merged**